.gitignore

```diff
@@ -1,4 +1,4 @@
 vendor/
 composer.lock
 tests/FacebookTestCredentials.php
-
```

Comment:
Done.

Thanks for the help, I did not know before about the global ignore or `.git\info\exclude` file

:+1: